assets-styleguide/js/styleguide.js

```diff
@@ -88,6 +88,13 @@ $(function () {
   }
 });
 
+//capture that the enter key was used to "click"
+$('.sidenav').on('keydown', 'a', function (e) {
+  if (e.which === 13) {
```

Comment:
13 should probably be a constant declared at the top of the function or file so people know what it is:

```
var ENTER = 13;
```